### PR TITLE
[OSDEV-1177] Update the release protocol to reflect the update of IAM roles for OpenSearch during the code freeze.

### PR DESCRIPTION
**Pull request by vladsha-dev**
[[OSDEV-1177](https://opensupplyhub.atlassian.net/browse/OSDEV-1177)] 
Since the terraform user requires access to OpenSearch to perform curl requests via AWS access keys, we need to manually add its ARN to the set of OpenSearch IAM roles for the preprod environment after creating this environment.

[OSDEV-1177]: https://opensupplyhub.atlassian.net/browse/OSDEV-1177?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: doc/release/RELEASE-PROTOCOL.md [hunk 1]
```diff
@@ -137,9 +137,10 @@ Make sure that:
...
+5. You need to run the `DB - Save Anonymized DB` workflow (if this job did not run on the same or the previous day). Once the Anonymized DB is successfully saved, run the `DB - Apply Anonymized DB` workflow to ensure that testing will be conducted with up-to-date data.
+6. In case there is a need to run a command in the terminal of the Django container, follow [this instruction](https://opensupplyhub.atlassian.net/wiki/spaces/SD/pages/140443651/DevOps+Guidelines+for+Migration+Database+Snapshots+and+ECS+Management#All-the-steps-described-in-this-Document-should-be-run-by-DevOps-or-Tech-Lead-Engineers-only%5BhardBreak%5D%5BhardBreak%5D%5BhardBreak%5D%5BhardBreak%5D%5BhardBreak%5D%5BhardBreak%5D%5BhardBreak%5DHow-to-correctly-run-migrations-for-our-four-environments%3F---Even-if-it-will-be-done-in-the-OSDEV-564-JIRA-ticket%2C-we-need-to-have-instructions-for-the-current-state-of-the-infrastructure.).
+7. Once the pre-prod environment is no longer needed for testing the release candidate, shut it down using the `Destroy Environment` GitHub workflow, as it is no longer required for the current version of the release.
```
Comment (VadimKovalenkoSNF) [inline doc/release/RELEASE-PROTOCOL.md:143]: This step is described in `Shut down the pre-prod environment` section.

**Review by VadimKovalenkoSNF — CHANGES_REQUESTED** [review]
Pls, align your notes with `Shut down the pre-prod environment` section.

**Review by roman-stolar — APPROVED** [review]

File: doc/release/RELEASE-PROTOCOL.md (hunk 1 above)
Comment (vladsha-dev) [reply doc/release/RELEASE-PROTOCOL.md]: I didn't add it but I have removed it.

**Review by roman-stolar — APPROVED** [review]
LGTM

**Review by VadimKovalenkoSNF — APPROVED** [review]

**Pull request merged**